include/annotated.hpp

```diff
@@ -131,10 +131,21 @@ struct Annotated {
         }
     }
 
+    template<typename U>
+    Annotated &
+    operator=(const U &sv) noexcept requires std::is_same_v<T, std::string> && std::is_same_v<U, std::string_view> {
```

Comment:
`&& std::is_same_v<U, std::string_view>`

I'd replace this with a `is_constructible_v<T, U>` and make `sv` perfectly-forwarded (maybe someone wants to use std::string instead of std::string_view, or something else that converts to string)